### DISCUSSION TITLE
Manage cars

### DISCUSSION THREAD

**Discussion by Wlad87** [teslamate-org/teslamate#3367] [Q&A]
Hello,
I already have the third Tesla in my account and unfortunately only the first two are displayed in the Grafana app and dr third is not. Since the first two are already sold, they can be deleted, but unfortunately I do not know how to do it. Can someone help me with this?
Currently the app is running on Synology and Docker.
Thank you and best regards

**Comment by cwanja** [comment]
Healthy discussion in #2719. You would need CLI access and commands are provided. Would specifically look at https://github.com/adriankumpf/teslamate/discussions/2719#discussioncomment-4551024. You would need to validate what IDs your cars are, but I assume they are `1` and `2`. 

Is the third car not displayed because it has no "custom" name? Believe there is an issue with when a name is missing, Grafana does not display the car.

**Reply by Wlad87** [reply]
can you tell me where to enter the commands?

![2023-10-16 21_26_38-OpenSSH SSH client](https://github.com/adriankumpf/teslamate/assets/148079892/c77aed04-de62-4a74-9bf4-1700224e4789)

**Reply by cwanja** [reply]
You will need to be in the command line where your Docker installation was made. The first commands require you to drop into your database through the command line. 

**Note:** always backup your data _before_ running commands altering the data. And stop the web service so that new data is not written at the same time.

**Reply by Wlad87** [reply]
I installed Teslamate on my Synology via Docker. How do I get into the directory? I don't even know where it was installed. I think I'm too stupid for this?!

**Reply by cwanja** [reply]
Not stupid at all, just will need assistance. 

From your image, which I did not see prior to my previous post - sorry about that - you will need to find the directory where TeslaMate is installed. I am not sure where that would be on Synology. Try looking in the `~` (home) directory for root, assuming that is the user that installed TeslaMate.

You have the right commands, you just need to be in the right directory. You could try running the `find` command to locate the YML file (`docker-compose.yml`). 

How do you start / stop TeslaMate on your NAS?

**Comment by alfahu27** [comment]
Hi I am Alfa, i think your readers will like our article about [car](https://news.olx.co.id). Maybe it could be an additional reference on your site?
